nav2_common/nav2_common/launch/parse_argument.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2023 LG Electronics.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from typing import Text
+
+class ParseArgument():
```

Comment:
Is this necessary? This has to be something in launch ros, right? How does every other launch param do this?

Reply:
Yeah, I agree what you thought. It looks kind of launch_ros part.

I can not say it's really necessary. 
But, as I mentioned above, could break a concept of Launch Argument so that it's hard to insist suggest(I guess).

Threfore, I just invented `ParseArgument` for usage in future work or other potential launch system.

If someone want parse an argument at time of building launch description like in this PR, just do like below.
```
from .parse_argument import ParseArgument
args = ParseArgument( arguement name what you want ).value()
```

BTW, if you think this part is not necessary. I can just move and merge into ParseMultiRobotPose.

Reply:
11

Reply:
>  But, as I mentioned above, could break a concept of Launch Argument so that it's hard to insist suggest(I guess).

I'm not sure what you mean. Couldn't this just _be_ a launch argument to grab the value? Then we use the `ParseMultiRobotPose` on that LaunchArgument output to get the "useful" parsed version (instead of the string)

Reply:
@hyunseok-yang this thread is still not closed. I don't think you should need to do this -- how do other launch configurations work? This feels like its reinventing something standard. 

>  Couldn't this just be a launch argument to grab the value? Then we use the ParseMultiRobotPose on that LaunchArgument output to get the "useful" parsed version (instead of the string)

Reply:
Yeah, looks like something standard.
I will just merge the code into ParseMultiRobot side.
And separate it as a future work on other repository.